<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainGUI</class>
 <widget class="QMainWindow" name="mainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbXVector">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>XVector</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>107</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblx">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbly">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblz">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leXVectorX"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leXVectorY"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leXVectorZ"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDefineVectorX">
        <property name="text">
         <string>Define</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbBaseVector">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>BaseVector</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>107</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="lblx_2">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbly_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblz_2">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leBaseVectorX"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leBaseVectorY"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leBaseVectorZ"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDefineBaseVector">
        <property name="text">
         <string>Define</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbYVector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>YVector</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>107</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="lblx_3">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbly_3">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblz_3">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leYVectorX"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leYVectorY"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leYVectorZ"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDefineVectorY">
        <property name="text">
         <string>Define</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="txtEditTextToWrite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>641</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWrite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnWrite</sender>
   <signal>clicked()</signal>
   <receiver>mainGUI</receiver>
   <slot>writeText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>writeText()</slot>
 </slots>
</ui>
